modules/concepts/services/_index.md

```diff
@@ -72,6 +72,24 @@ services:
       - "My custom message"
 ```
 
+NOTE : Since 8.x it is possible to load PHP / XML files for modules services
```

Comment:
```suggestion
{{% notice tip %}}Since version 8.0 of PrestaShop it is possible to load PHP / XML files for modules services{{% /notice %}}
```

what do you think?

Reply:
Is it needed to mention "Since 8.x" ? `8.x` branch is the branch for PS 8 doc 😄

Reply:
LGTM ... Thanks guy !